config/settings.py

```diff
@@ -406,3 +408,9 @@
         },
     },
 }
+
+# debug tool bar
+if DEBUG:
+    import socket
+    hostname, _, ips = socket.gethostbyname_ex(socket.gethostname())
+    INTERNAL_IPS = [ip[: ip.rfind(".")] + ".1" for ip in ips] + ["127.0.0.1"]
```

Comment:
Does this work?

Reply:
@thenav56 Yes it works , I have been using this doc https://django-debug-toolbar.readthedocs.io/en/latest/installation.html

Reply:
Let's add this as a note there https://django-debug-toolbar.readthedocs.io/en/latest/installation.html#configure-internal-ips